src/css/categories.css

```diff
@@ -70,16 +64,16 @@ span.category-card-title {
   color: rgba(244, 244, 244, 0.4);
 }
 
-/* Styles for Tablet devices */
-@media only screen and (min-width: 768px) {
+@media only screen and (max-width: 375px) {
```

Comment:
with media we are using min-width for mobile wirst approach, my comments was regarding max-width of the .container